flexmeasures/ui/templates/base.html

```diff
@@ -51,13 +51,17 @@
     {% block body %}
 
     {% block nav %}
+    <style>
+        .white-color {
```

Comment:
I believe using CSS classes only for a color is bad style.

Use `color: var(--white);`
Check ui/static/css/flexmeasures.css to see how colors are defined in this project.

Reply:
i have corrected this as well